ckanext/canada/schemas/presets.yaml

```diff
@@ -2755,25 +2752,64 @@ presets:
         fr: Séparé de
       value: separated_from
     - label:
-        en: Split into ...
-        fr: Divisé en …
+        en: Split into
+        fr: Divisé en
       value: split_into
     - label:
-        en: Merged with ...
+        en: Merged with
         fr: Fusionné avec
       value: merged_into
     - label:
         en: Changed back to
         fr: Restauré à
       value: changed_back_to
+    form_include_blank_choice: true
     form_snippet: select.html
     display_snippet: select.html
     validators: scheming_required scheming_choices
+    required: true
+    form_panel: relationships
+
+# Field = Related Record URL.
+- preset_name: canada_dataset_related_url
+  values:
+    field_name: related_url_translated
+    label:
+      en: Related Record URL
+      fr: FR Related Record URL FR
+    help_text:
+      en: The URL for online access to the related record
+      fr: FR The URL for online access to the related record FR
+    fluent_form_label:
+      en:
+        en: "Related Record URL (English)"
+        fr: "FR Related Record URL (anglais) FR"
+      fr:
+        en: "Related Record URL (French)"
+        fr: "FR Related Record URL (français) FR"
+    fluent_help_text:
+      en:
+        en: The URL for online access to the related record
+        fr: FR The URL for online access to the related record FR
+      fr:
+        en: The URL for online access to the related record
+        fr: FR The URL for online access to the related record FR
+    # copied from fluent_text preset
+    form_snippet: fluent_text.html
+    display_snippet: fluent_link.html
+    display_attributes:
+      style: "word-wrap: break-word"
+    error_snippet: fluent_text.html
+    validators: fluent_text
+    output_validators: fluent_core_translated_output
```

Comment:
not a core field so this isn't the right validator

Reply:
just use `fluent_text_output`?